src/utils/pxls/websocket_client.py

```diff
@@ -41,7 +42,15 @@ async def _listen(self):
 
         while True:
             pxls_validate = str(uuid.uuid4())
-            headers = {"Cookie": f"pxls-validate={pxls_validate}"}
+            if self.pxls_cfauth:
+                headers = {
+                    "Cookie": f"pxls-validate={pxls_validate}",
+                    "x-pxls-cfauth": f"{self.pxls_cfauth}"
+                }
+            else:
+                headers = {
+                    "Cookie": f"pxls-validate={pxls_validate}"
+                }
```

Comment:
Instead of assigning headers twice, you should assign it once with `Cookie` as the default item, then add `x-pxls-cfauth` if using `self.pxls_cfauth`.

Reply:
I'll take note of this, thanks!